webapp/src/main/java/io/github/microcks/web/KeycloakConfigController.java

```diff
@@ -40,10 +40,10 @@
    private static Logger log = LoggerFactory.getLogger(KeycloakConfigController.class);
 
 
-   @Value("${keycloak.auth-server-url}")
+   @Value("${external_url}")
    private final String keycloakServerUrl = null;
 
-   @Value("${keycloak.realm}")
+   @Value("${external_realm}")
```

Comment:
What is the reason for also providing an external realm name ? This could not be different as the realm name used by adapte otherwise it won't work no ?